frontend/web/components/AdminAPIKeys.js

```diff
@@ -224,11 +224,10 @@ export default class AdminAPIKeys extends PureComponent {
         {!!apiKeys && !!apiKeys.length && (
           <PanelSearch
             className='no-pad'
-            title='API Keys'
             items={apiKeys}
             header={
               <Row className='table-header'>
-                <Flex className='table-column px-3'>API Keys</Flex>
+                <Flex className='table-column px-3'>Terraform API Keys</Flex>
```

Comment:
Why did this change back to Terraform API keys? This was changed on purpose since we're soon going to support regular API keys (using roles etc.)

Reply:
It's clearer right now. We would probably want to split out legacy keys in a separate table anyway?

Reply:
What do you mean by 'legacy keys' ??

Reply:
Ah I mean I'm not sure what we are doing with "Terraform keys" are we going to deprecate them or call them something different?

Reply:
They're just going to exist in the same 'API keys' section

Reply:
It's the exact same entity under the hood.